src/components/pages/Home/HomeContainer.js

```diff
@@ -28,6 +28,8 @@ function HomeContainer({ LoadingComponent }) {
     return () => (isSubscribed = false);
   }, [memoAuthService]);
 
+  console.log(userInfo);
```

Comment:
Console.logs() should not be left  in the code

Reply:
Will remove for next commit, thank you for the catch!